2-virtual-machines-containers/2.1-golden-images.md

```diff
@@ -5,14 +5,35 @@
 ![](img2/linux_light.svg ':size=100x100 :class=light-mode-icon')
 ![](img2/linux_dark.svg ':size=100x100 :class=dark-mode-icon')
 
+?> If you have an m1 mac checkout virtualbox beta build for arm64. ( Note that as of 23/02/2023 the virtualbox beta build for arm64 did not work)
+
+?> assuming you have an m1 mac & the virtualbox beta is still not operational, the only m1 compatible section is the compatibility section.
+
+### M1 Compatibility Excersize
+
+1. You will need to install the utm project. assuming you have brew  `brew install --cask utm` should work
```

Comment:
```suggestion
1. You will need to install the UTM project. assuming you have brew  `brew install --cask utm` should work
```

Reply:
Can we add a link to the UTM project here also for curious apprentices who want to know _what_ they are installing